docs/design/specs/Ecma-335-Augments.md

```diff
@@ -1020,3 +1021,7 @@ A checked user-defined operator is expected to throw an exception when the resul
 Section "I.12.6.6 Atomic reads and writes" adds clarification that the atomicity guarantees apply to built-in primitive value types and pointers only.
 
 A conforming CLI shall guarantee that read and write access of *built-in primitive value types and pointers* to properly aligned memory locations no larger than the native word size (the size of type native int) is atomic (see §I.12.6.2) when all the write accesses to a location are the same size.
+
+## Backward branch constraints
+
+Section "II.1.7.5 Backward branch constraints" is deleted. These constraints were not enforced by any mainstream .NET runtime and they are not respected by .NET compilers. It means that it is not possible to infer the exact state of the evaluation stack at every instruction with a single forward-pass through the CIL instruction stream.
```

Comment:
I think Roslyn C# compiler was conforming to "one-pass". At least it used to conform when its backend/IL-generator was written. Things may have changed since then though.